braid/hierarchy.c

```diff
@@ -173,8 +174,17 @@ _braid_InitHierarchy(braid_Core    core,
       }
       else
       {
-         /* This is the coarsest level */
-         if ( (level > 0) || (!refined) )
+         
+         /* If solving coarsest grid by relaxation, initialize a standard coarse grid*/
+         if( relax_only_cg && (gclower < gcupper) )
```

Comment:
@rfalgout Hi Rob, This is the spot in hierarchy.c that I tweaked to create a normal C/F splitting on the coarsest grid.  Take a look at the logic, if you have a moment, to double check.